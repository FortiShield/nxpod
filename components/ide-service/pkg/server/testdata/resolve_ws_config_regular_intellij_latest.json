{
  "context": "{\"isFile\":false,\"path\":\"\",\"title\":\"nxpkg/empty \",\"revision\":\"\",\"repository\":{\"cloneUrl\":\"https://github.com/nxpkg/empty.git\",\"host\":\"github.com\",\"name\":\"empty\",\"owner\":\"nxpkg\",\"private\":false},\"normalizedContextURL\":\"https://github.com/nxpkg/empty\",\"checkoutLocation\":\"empty\"}",
  "ide_settings": "{\"settingVersion\":\"2.0\",\"defaultIde\":\"intellij\",\"useLatestVersion\":true}",
  "workspace_config": "{\"ports\":[],\"tasks\":[],\"image\":\"docker.io/nxpod/workspace-full:latest\",\"_origin\":\"default\"}"
}
