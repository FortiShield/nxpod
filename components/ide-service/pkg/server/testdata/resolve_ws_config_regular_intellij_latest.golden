{
    "Resp": {
        "envvars": [
            {
                "name": "NXPOD_IDE_ALIAS",
                "value": "intellij"
            }
        ],
        "supervisor_image": "ghcr.io/nxpkg/build/supervisor:commit-ff38b98b7dde4929159bcaeec68d178898dc2139",
        "web_image": "ghcr.io/nxpkg/build/ide/code:nightly@sha256:8669f6ab8dceefde11138bf4bc0ee73b934b2cc47e6135426f05fb4343ec5ab1",
        "ide_image_layers": [
            "ghcr.io/nxpkg/build/ide/intellij:latest@sha256:e07524e52089829dc8d3b38f7d18fb51b24f07aed7d8e4e6e447899687978d43",
            "ghcr.io/nxpkg/build/ide/jb-backend-plugin:commit-b38092639d1783a1957894ddd4f492b3cdc9794a-latest",
            "ghcr.io/nxpkg/build/ide/jb-launcher:commit-b38092639d1783a1957894ddd4f492b3cdc9794a"
        ],
        "ide_settings": "{\"defaultIde\":\"intellij\",\"useLatestVersion\":true}\n"
    },
    "Err": ""
}
